src/browser/win-registry.coffee

```diff
@@ -0,0 +1,62 @@
+path = require 'path'
+Spawner = require './spawner'
+
+if process.env.SystemRoot
+  system32Path = path.join(process.env.SystemRoot, 'System32')
+  regPath = path.join(system32Path, 'reg.exe')
+else
+  regPath = 'reg.exe'
+
+# Registry keys used for context menu
+fileKeyPath = 'HKCU\\Software\\Classes\\*\\shell\\Atom'
+directoryKeyPath = 'HKCU\\Software\\Classes\\directory\\shell\\Atom'
+backgroundKeyPath = 'HKCU\\Software\\Classes\\directory\\background\\shell\\Atom'
+applicationsKeyPath = 'HKCU\\Software\\Classes\\Applications\\atom.exe'
+
+# Spawn reg.exe and callback when it completes
+spawnReg = (args, callback) ->
+  Spawner.spawn(regPath, args, callback)
+
+# Install the Open with Atom explorer context menu items via the registry.
+#
+# * `callback` The {Function} to call after registry operation is done.
+#   It will be invoked with the same arguments provided by {Spawner.spawn}.
+#
+# Returns `undefined`.
+exports.installContextMenu = (callback) ->
```

Comment:
We should break these up into installContextMenuForFolder, installContextMenuForFile and installFileHandler but that can be done as part of the subsequent options work.